### PR TITLE
Refacto some sensible selectors and add some missing selectors to classic selector map

### LINKED ISSUE #25583: Dangerous JS selectors should be removed
There is a lot of potentially dangerous JS selectors. We shouldn't be scared of breaking stuff when redoing the markup.

Selectors using `.row`, `.col-XXX`, `.container`, hardcoded boostrap markup into JS.

### Example
this.productsGrid = `${this.contentDiv} > div.row > div > div.row:nth-of-type(2)`;

### File list
**Admin**
https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/default/js/bundle/product/form.js
https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/default/js/bundle/product/form.js
https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/new-theme/js/pages/product/product-map.js
https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.ts

**Tests**
https://github.com/PrestaShop/PrestaShop/blob/develop/tests/UI/pages/BO/catalog/brands/view.js
https://github.com/PrestaShop/PrestaShop/blob/develop/tests/UI/pages/BO/catalog/products/index.js
https://github.com/PrestaShop/PrestaShop/blob/develop/tests/UI/pages/BO/catalog/suppliers/view.js

**Theme**
https://github.com/PrestaShop/PrestaShop/blob/develop/themes/_core/js/selectors.js
https://github.com/PrestaShop/PrestaShop/blob/develop/themes/_core/js/product.js
https://github.com/PrestaShop/PrestaShop/blob/develop/themes/classic/_dev/js/cart.js

#### Additional information
* PrestaShop version: develop

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Some selectors were too sensible and some were missing in classic theme
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #25583.
| How to test?      | Test if every pages concerned are working fine (View order with add product feature, View supplier, View brand, Classic theme cart) and run tests of Suppliers and Brands
| Possible impacts? | Global

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25595)

### CODE REVIEW

**Review by matthieu-rolland — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Review by sowbiba — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Comment by hibatallahAouadni** [comment]
Ping @NeOMakinG could you please resolve the Conflicting files :pray: 
Thanks in advance!

**Review by atomiix — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Comment by khouloudbelguith** [comment]
Hi @NeOMakinG,

thanks to check the Conflicting files.

Thank you!

File: src/PrestaShopBundle/Resources/views/\ [hunk 1]
```diff
@@ -0,0 +1,115 @@
+{#**
```
Comment (Progi1984) [inline src/PrestaShopBundle/Resources/views/\]: @NeOMakinG It seems the file has a bad file name.

**Review by Progi1984 — CHANGES_REQUESTED** [review]
@NeOMakinG :warning: A file has a bad file name

**Review by atomiix — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by khouloudbelguith** [comment]
Hi @NeOMakinG,

It is ok :heavy_check_mark: 
1. Classic theme cart:
- I check a product with customization (file or text)
- Increase/Decrease quantity => ok (a product will be out of stock / minimal quantity)
- Add a voucher / remove a voucher (with code & node_code)
- Remove a product
https://watch.screencastify.com/v/2OSGeHQBmFYWq7O0XuXD

2. run tests of Suppliers and Brands => ok 
[Mochawesome Report_brands&suppliers.pdf](https://github.com/PrestaShop/PrestaShop/files/7713573/Mochawesome.Report_brands.suppliers.pdf)

3. View order with add product feature
- I tried to add products to order (all types)
- Create an invoice when adding a product
- Add a product with a specific cart rule

=> automatic tests ok also 
image


4. View brand => ok and I reported this issue: https://github.com/PrestaShop/PrestaShop/issues/26985
5. View supplier  => ok
image


Thanks!

**Comment by Progi1984** [comment]
Thanks @NeOMakinG & @khouloudbelguith

**Pull request merged**